apps/spruce/src/components/Spawn/spawnHostModal/getFormSchema.tsx

```diff
@@ -104,158 +114,8 @@ export const getFormSchema = ({
-                      default: "",
-                      minLength: 1,
-                    },
-                    savePublicKey: {
-                      title: "Save Public Key",
-                      type: "boolean" as "boolean",
-                      default: false,
-                    },
-                  },
-                  dependencies: {
-                    savePublicKey: {
-                      oneOf: [
-                        {
-                          properties: {
-                            savePublicKey: {
-                              enum: [false],
-                            },
-                          },
-                        },
-                        {
-                          properties: {
-                            savePublicKey: {
-                              enum: [true],
-                            },
-                            newPublicKeyName: {
-                              title: "Key name",
-                              type: "string" as "string",
-                              default: "",
-                              minLength: 1,
-                            },
-                          },
-                        },
-                      ],
-                    },
-                  },
-                },
-              ],
-            },
-          },
-        },
-        expirationDetails: {
-          title: "Expiration Details",
-          type: "object" as "object",
-          properties: {
-            noExpiration: {
-              default: false,
-              type: "boolean" as "boolean",
-              title: "",
-              oneOf: [
-                {
-                  type: "boolean" as "boolean",
-                  title: "Expirable Host",
-                  enum: [false],
-                },
-                {
-                  type: "boolean" as "boolean",
-                  title: "Unexpirable Host",
-                  enum: [true],
-                },
-              ],
-            },
-          },
-          dependencies: {
-            noExpiration: {
-              oneOf: [
-                {
-                  properties: {
-                    noExpiration: {
-                      enum: [false],
-                    },
-                    expiration: {
-                      type: "string" as "string",
-                      title: "Expiration",
-                      default: getDefaultExpiration(),
-                      minLength: 6,
-                    },
-                  },
-                },
-                {
-                  properties: {
-                    noExpiration: {
-                      enum: [true],
-                    },
-                    ...(!isProduction() && { hostUptime: hostUptime.schema }),
-                  },
-                },
-              ],
-            },
-          },
-        },
+        publicKeySection: publicKeys.schema,
+        expirationDetails: expirationDetails.schema,
```

Comment:
I think I can't spawn a host, it keeps giving me this error:
<img width="711" alt="Screenshot 2024-05-06 at 5 37 45 PM" src="https://github.com/evergreen-ci/ui/assets/47064971/22006858-2972-40bd-ba6a-f184b947f212">